EXPLORATION.md

```diff
@@ -0,0 +1,52 @@
+Support for Power Pressure
+===
+
+This we need to do to create a Power Pressure Observer POC.
```

Comment:
```suggestion
This is what we need to in order to to create a Power Pressure Observer POC.
```

Reply:
the syntax is not correct.
- "what we need to " not sure if you forgot the "do", it s understandable without.
- "in order to to" twice "to"

Reply:
What do you mean :D ???

Reply:
hoo, somehow it showed it in a very strange way! Now it looks fine.
Sorry that is fine!